### PR TITLE
The HyperLinks to About and Privacy docs on Portability Analyzer Setting are not navigatable

### PR DESCRIPTION
**Pull request by Lxiamail**
…Analyzer Setting are not navigatable

### CODE REVIEW

File: src/ApiPort/ApiPort.VisualStudio/Views/OptionsPageControl.xaml [hunk 1]
```diff
@@ -186,7 +186,7 @@
 
             <TextBlock Grid.Column="2"
                        HorizontalAlignment="Right">
-                <Hyperlink RequestNavigate="NavigateToMoreInformation"
+                <Hyperlink NavigateUri="DocumentationLinks.About" RequestNavigate="NavigateToMoreInformation"
```
Comment (twsouthwick) [inline src/ApiPort/ApiPort.VisualStudio/Views/OptionsPageControl.xaml]: Is this needed? Looks like you either need to set the uri or could consolidate the event handlers

**Comment by twsouthwick** [comment]
If you reference the issue in the body of the PR, it'll create a link. ie

#764

File: src/ApiPort/ApiPort.VisualStudio/Views/OptionsPageControl.xaml (hunk 1 above)
Comment (Lxiamail) [reply src/ApiPort/ApiPort.VisualStudio/Views/OptionsPageControl.xaml]: Yes, adding the uri actually fixed the original problem. Looks like consolidate the event handler only is not enough.

**Comment by twsouthwick** [comment]
And if you say `Fixes ...` it'll automatically close the issue when merged

**Pull request merged**